backend/grading-service/src/models/its/error-feedback.ts

```diff
@@ -1,4 +1,4 @@
 export type ErrorFeedback = {
-  lineNumber: number;
-  hintStrings: string[];
+  line: number;
+  hints: string[];
 };
```

Comment:
Ah so the change was done here

Reply:
Paiseh, forgot to push this changes to the previous PR xD